### DISCUSSION TITLE
Welcome to tts.bot Discussions!

### DISCUSSION THREAD

**Discussion by johnebgood** [security-live/tts.bot#8] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Use this forum for feature requests.
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

If you find a security issue please email security@securitylive.com directly so we can fix it properly before disclosing to the public.

**Comment by Dylan2Royle** [comment]
WEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEE
